<launch>
  <arg name="base" default="true" />  

  <node name="segbot_arm_joystick_node" pkg="segbot_arm_joystick" type="segbot_arm_joystick" output="screen" >
     <param name="base" value="$(arg base)" type="bool" />
  </node>
  <node name="joy" pkg="joy" type="joy_node" />
  <node name="speak_message_service" pkg="bwi_services" type="speak_message_service" >
     <param name="speed" value="160" type="int" />
     <param name="voice" value="default" type="str" />
     <param name="pitch" value="50" type="int" />
  </node>
</launch>
  




